pkg/azuredisk/azuredisk.go

```diff
@@ -608,7 +608,7 @@ func (d *DriverCore) getUsedLunsFromNode(ctx context.Context, nodeName types.Nod
 }
 
 // getNodeInfoFromLabels get zone, instanceType from node labels
-func getNodeInfoFromLabels(ctx context.Context, nodeName string, kubeClient clientset.Interface) (string, string, error) {
+func GetNodeInfoFromLabels(ctx context.Context, nodeName string, kubeClient clientset.Interface) (string, string, error) {
```

Comment:
Consider moving `GetNodeInfoFromLabels` to `pkg/util/util.go` since this general utility function is now accessed outside of this package.